apps/api/src/app/layouts/use-cases/create-layout/create-layout.use-case.ts

```diff
@@ -43,4 +47,27 @@ export class CreateLayoutUseCase {
       isDeleted: layout.deleted,
     };
   }
+
+  private getExtractedVariables(variables: ITemplateVariable[], content: string): ITemplateVariable[] {
+    const contentService = new ContentService();
```

Comment:
What's our internal take on the use of a service? Just asking to consider suggesting to add it to the constructor of the use case rather that instantiate it every time the method is being called.

Reply:
Yes, I was also wondering what will be better